layouts/fastly/helix.vcl

```diff
@@ -1101,6 +1101,19 @@ sub hlx_fetch_query {
   }
 }
 
+sub hlx_fetch_lnk {
+  if (req.http.X-Request-Type == "Content/LNK") {
+    if (!req.is_esi_subreq) {
+      set beresp.http.X-PostFetch = beresp.http.X-PostFetch + "; hlx_fetch_lnk";
+    }
+    if (req.http.X-Orig-URL ~ "hlx_report=true") {
+      set beresp.http.Access-Control-Allow-Origin = req.http.Origin;
+      set beresp.http.Access-Control-Allow-Methods = "GET, OPTIONS, POST";
```

Comment:
```suggestion
      set beresp.http.Access-Control-Allow-Methods = "GET, OPTIONS";
```
`POST` is not needed for .lnk requests

Reply:
I would remove all this and just allow the CORS headers from the backend to be honoured.